content/docs/main/traffic-management/service-entry.md

```diff
@@ -0,0 +1,316 @@
+---
```

Comment:
This should go under the Istio integration section instead of traffic-management.